File: backend-php/Models/CityInfo.php
<?php

class CityInfo {
    public string $name = 'Ortega';
    public string $department = '';
    public string $description = '';
    public string $history = '';
    public int $population = 0;
    public float $area = 0;
    public int $altitude = 0;
    public float $temperature = 0;
    public string $climate = '';
    public string $economy = '';
    public string $culture = '';
    public array $highlights = [];
    public string $location = '';
    
    public function toArray(): array {
        return [
            'name' => $this->name,
            'department' => $this->department,
            'description' => $this->description,
            'history' => $this->history,
            'population' => $this->population,
            'area' => $this->area,
            'altitude' => $this->altitude,
            'temperature' => $this->temperature,
            'climate' => $this->climate,
            'economy' => $this->economy,
            'culture' => $this->culture,
            'highlights' => $this->highlights,
            'location' => $this->location
        ];
    }
}

File: backend/Models/CityInfo.cs
namespace OrtegaTourism.Models
{
    public class CityInfo
    {
        public string Name { get; set; } = "Ortega";
        public string Description { get; set; } = string.Empty;
        public string History { get; set; } = string.Empty;
        public int Population { get; set; }
        public string Climate { get; set; } = string.Empty;
        public List<string> Highlights { get; set; } = new List<string>();
        public string Location { get; set; } = string.Empty;
    }
}